### ISSUE TITLE
Wildcard for content file matching not working

### ISSUE THREAD

**Issue by occupant** [FullHuman/gulp-purgecss#1]
Hi there,

I'm using gulp-purgecss 0.12.0 and I have an issue with the following configuration:

```
gulp.task('purgecss', function() {
  return gulp.src(paths.build.css + 'styles-all.min.css')
    .pipe(purgecss({
      content: [paths.build.html + '*.html']
    }))
    .pipe(gulp.dest(paths.dist.css))
});
```

What I would expect is that all of my html files in the html build directory are parsed to generate the purged output in the purgecss destination directory. Instead, I only see any root values I've included in my css  (ie. `:root { someproperty: somevalue; }` )

If I change my gulp task to parse a single html file, the purgecss task works as expected:

```
gulp.task('purgecss', function() {
  return gulp.src(paths.build.css + 'styles-all.min.css')
    .pipe(purgecss({
      content: [paths.build.html + 'home.html']
    }))
    .pipe(gulp.dest(paths.dist.css))
});
```

**Comment by Ffloriel** [comment]
Thank you for reporting this issue.
I will make a fix soon. In the meantime, as a workaround, you should be able to do it with:

```js
const glob = require('glob')

gulp.task('purgecss', function() {
  return gulp.src(paths.build.css + 'styles-all.min.css')
    .pipe(purgecss({
      content: glob.sync(paths.build.html + '*.html')
    }))
    .pipe(gulp.dest(paths.dist.css))
});

```

**Comment by Ffloriel** [comment]
It should be fix in v0.13.0.
Let me know if you find any issue

**Comment by icarlosmendez** [comment]
Hi Ffloriel,
I know you have closed this issue but I seem to be having the same problem described by "Occupant". Could you let me know if you see any obvious errors in my code. As described above, if I set a single file in the content section, I get results. An array of individual files also produces results, but the use of the wildcard characters results in an empty stream and ultimate failure. Any guidance will be welcome. Thanks.

`gulp.task('purgecss', () => {
    return gulp.src('css/build/compiled.css')
        .pipe(purgecss({
            content: ['../views/**/*']
        }))
        .pipe(rename('purged.css'))
        .pipe(gulp.dest('css/build'))
});`
